modules/nw-dpu-installing-operator.adoc

```diff
@@ -0,0 +1,162 @@
+// Module included in the following assemblies:
+//
+// * networking/networking_operators/installing-dpu-operator.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-dpu-operator_{context}"]
+= Installing the DPU Operator
+
+The DPU Operator on the host cluster side is responsible for the life-cycle management of the `ovn-kube` components and the necessary host network initialization on IPU.
+
+As a cluster administrator, you can install the DPU Operator by using the {product-title} CLI or the web console.
```

Comment:
Note that this has to be repeated on all the clusters in the topology, at least on the host side and once for each DPU.